### PR TITLE
Bug 1986757: Set timeoutSeconds for keepalived liveness probe

### PR DESCRIPTION
**Pull request by cybertron**
timeoutSeconds defaults to 1 second, but our liveness probe attempts
to wait for 5 seconds. This seems to be causing frequent liveness
probe timeouts in some environments which triggers unnecessary
restarts.

This patch sets timeoutSeconds to 5 to match the expectations of
the probe.

**- Description for the changelog**
Fixed an issue with the keepalived liveness probe that could cause spurious timeouts.

### CODE REVIEW

**Review by mandre — COMMENTED** [review]
/lgtm

**Comment by cybertron** [comment]
All of the affected platforms are now passing. This is a pretty simple change that isn't platform-specific so I think it could probably go in (although it would be even better if someone from the remaining two platforms could chime in).

**Review by kikisdeliveryservice — APPROVED** [review]
Adding an approval and also a hold to be removed by @cybertron once he feels it's ready and has necessary approvals.

/hold

**Comment by yboaron** [comment]
@cybertron  I think we can remove the hold and merge this PR

**Comment by cybertron** [comment]
/hold cancel

Agreed, thanks.

**Comment by cybertron** [comment]
The failing jobs all appear to be permafails and none of them even deploy the changed template. I think we just need to skip them.

**Pull request merged**
